triple-store/triple-store-impl-rdf4j/src/main/java/com/powsybl/triplestore/impl/rdf4j/TripleStoreRDF4J.java

```diff
@@ -182,6 +190,38 @@ public PropertyBags query(String query) {
         return results;
     }
 
+    @Override
+    public void copyFrom(TripleStore source) {
+        Repository sourceRepo = ((TripleStoreRDF4J) source).getRepository();
```

Comment:
Same remark

Reply:
we've added the assert on source and check for the object type before casting